internal/operations/operations.go

```diff
@@ -22,36 +22,48 @@ import (
 
 var (
 	Operations = adapter.Operations{
-		config.InstallConsulCommand: &adapter.Operation{
+		config.Consul182DemoOperation: &adapter.Operation{
```

Comment:
Do we have any way to avoid hardcoding version into these charts?

Reply:
There are quite a few aspects to consider regarding this. For instance the need for version specific configuration different from the vendors default config, handling Helm chart versions that differ from service mesh versions, activating and configuring new released features, etc. It probably is different for different adapters. Would be great to discuss this in a call or dev meeting.